app/store/modules/product/index.js

```diff
@@ -0,0 +1,35 @@
+// store/modules/product/index.js
+import productData from './mock';
+
+const namespaced = true;
+
+// initial state
+const state = {
+  items: productData // demo
+};
+
+// getters
+const getters = {
+  /**
+   * Gets a product
+   *
+   * @param   {state}   Current state
+   * @returns  {Array} Product
+   */
+  // TODO handle state error e.g. (eslint no-shadow: ["error", { "allow": ["state"] }])
```

Comment:
- [ ] This is not an option. It's bad practice to have the same variable names on different "levels" and could lead to unexpected behaviour. I personally would recommend that we skip the variable part and put all sections directly in the export object. The only other possibility is to find an alternative variable name on the method or the state which I think is not a nice solution.